content/sensu-enterprise/3.1/integrations/ec2.md

```diff
@@ -102,14 +108,33 @@ type           | Array
 allowed values | `ok`, `warning`, `critical`, `unknown`
 example        | {{< highlight shell >}} "severities": ["critical", "unknown"]{{< /highlight >}}
 
-timeout      | 
+accounts     | 
 -------------|------
-description  | The handler execution duration timeout in seconds (hard stop).
+description  | List of trusted client names and roles? See [`name` and `role_arn` attributes][7].
 required     | false
-type         | Integer
-default      | `10`
-example      | {{< highlight shell >}}"timeout": 30{{< /highlight >}}
+type         | Array of hashes
+example      | {{< highlight shell >}}"accounts": [
+  {
+    "name": "sensuapp",
+    "role_arn": "arn:aws:iam::921689585014:role/CrossAccountSignin"
+  }
+]{{< /highlight >}}
+
+#### `accounts` attributes
 
+name         | 
+-------------|------
+description  | AWS account name?
+required     | false
+type         | String
+example      | {{< highlight shell >}}"name": "sensuapp"{{< /highlight >}}
+
+role_arn     | 
+-------------|------
+description  | Amazon resource role names?
```

Comment:
ARNs are [Amazon Resource Names](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) , kind of like a URL for a resource in Amazon.